src/cli_util.rs

```diff
@@ -890,7 +892,16 @@ impl WorkspaceCommandHelper {
     }
 
     pub fn start_transaction(&self, description: &str) -> Transaction {
-        let mut tx = self.repo.start_transaction(&self.settings, description);
+        Self::start_transaction_helper(&self.repo, &self.settings, &self.string_args, description)
+    }
+
+    pub fn start_transaction_helper(
```

Comment:
Nit: I'm split on this name, but don't have any good suggestions. `WorkspaceCommandHelper::start_transaction_helper` is quite a mouthful.

Reply:
Made this a free function